examples/03-tips-n-tricks/03-using_inline_functions_and_Query.py

```diff
@@ -0,0 +1,116 @@
+"""
+.. _ref_how_to_use_Query:
```

Comment:
```suggestion
.. _ref_how_to_use_query:
```
Let's keep these consistently as all lowercase.